### PR TITLE
Layout update 

### PR DESCRIPTION
**Pull request by shahab-ramzan**
tab icon added, networkcall code refactored, change header on user's log in/out

### CODE REVIEW

File: src/components/common/headContentAndPermisisonCheck/headContent.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ function HeadContent({
         }
       />
       <link
-        // href={`${process.env.IMAGES_BASE_URL}/static-content/canonizerLogo.png.webp`}
+        href="https://canonizer.com/img/logo.png"
```
Comment (akuks) [inline src/components/common/headContentAndPermisisonCheck/headContent.tsx]: Probably put somewhere under the public folder. And then access it using the `BASE_URL/images/logo.png`. Moreover its pointing to old logo

File: src/network/networkCall.ts [hunk 2]
```diff
@@ -1,42 +1,36 @@
 import axios from "axios";
 import K from "../constants";
-import { message } from "antd";
+// import { message } from "antd";
```
Comment (akuks) [inline src/network/networkCall.ts]: Do we need this in future?

File: src/network/networkCall.ts [hunk 3]
```diff
@@ -1,42 +1,36 @@
...
 export default class NetworkCall {
   static async fetch(request, useLoading = true) {
+    const axiosCall = () => {
+      return NetworkCall.axios({
+        method: request.method,
+        url: request.url,
+        data: request.body,
+        headers: request.headers,
+        validateStatus: (status) => {
+          return (status >= 200 && status < 300) || status === 304;
```
Comment (akuks) [inline src/network/networkCall.ts]: Is there any status between 200 and 300?

**Review by akuks — CHANGES_REQUESTED** [review]
Please go through the requested changes

File: src/components/common/headContentAndPermisisonCheck/headContent.tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/components/common/headContentAndPermisisonCheck/headContent.tsx]: Exactly, I already have tried. Need discussion on it

**Review by akuks — APPROVED** [review]

**Pull request merged**